config/fxdata/magic.cfg

```diff
@@ -1350,6 +1353,32 @@ WithstandHitAgainst = WALL DOOR
 Properties = REBOUND_IMMUNE NO_HIT NO_STUN WIND_IMMUNE EXPLODE_FLESH
 UpdateLogic = 7
 
+[shot33]
```

Comment:
mh a new shot will break mappack or campaign that made a shot33 and so their fart will use their shot 🤔?

Reply:
I don't see how else it can be done.

Reply:
> I don't see how else it can be done.

well you can keep the function creature_fart and let it that way for the [instance29] then it's not a problem if [shot33] exist for a map it won't break Fart 🤔

Reply:
That defeats the purpose of this PR, as everything in creature_fart is hardcoded. That's why I'm getting rid of it.